add note about previous versions of these docs (#227)
smartdevicelink/sdl_hmi_integration_guidelines: docs/Overview/index.md

|TTS|Text To Speech|
|UTC|Universal Time Coordinate|
|VDOP|Vertical Dilution of Precision|
|VR|Voice Recognition|

## Previous versions of these docs

> This documentation is transpiled from markdown in the [sdl_hmi_integrations GitHub repository](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/) to HTML. If you wish to view these guides as they were in the past you can navigate to [the releases page](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/releases) to view or download the markdown in a previous state.

Reviewer: ```suggestion
This documentation is transpiled from markdown in the [sdl_hmi_integrations GitHub repository](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/) to HTML. To view the guides for previous releases, you can navigate to the [releases page](https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/releases) to view or download the markdown files.
```
